src/core/admin.sol

```diff
@@ -72,22 +67,22 @@ contract Admin {
         uint loan = admit.admit(registry, nft, principal, usr);
         appraiser.file(loan, appraisal);
 
-        (,, uint speed,) = debtRegister.rates(rate);
+        (,, uint speed,) = pile.rates(rate);
```

Comment:
the whitelist method should be removed

Reply:
👍https://github.com/centrifuge/tinlake/issues/168